Reason: We ship pycryptodome as a full replacement of pycrypto, so the
dependency is on pycryptodome, not pycryptodomex
Upstream: No
--- a/requirements/crypto.txt
+++ b/requirements/crypto.txt
@@ -1 +1 @@
-pycryptodomex>=3.9.8
+pycryptodome>=3.9.8
